readcourse.data

```diff
@@ -0,0 +1,148 @@
+import json
+import os
+from pymongo import MongoClient
+
+class Course:
+	def __init__(self, crn):
+		self.crn = crn
+
+	def __init__(self, **kwargs, *args):
+		self.UID = kwargs.['UID']
```

Comment:
I don't think this is right.
`kwargs` is a Dict. That means you should put `kwargs['UID']`
In addictional, `kwargs` should be assigned to another one of meaningful variable

Reply:
Sorry I don't understand the third line?

Reply:
should be `some_variable_name = kargs`